apps/11gist-blog/script.js

```diff
@@ -2,4 +2,44 @@
 
 $(document).ready(function() {
   // You code here
+  $.ajax('https://api.github.com/users/matthew-s-howard/gists', {
+    success: function(posts) {
+      posts.forEach(function(post) {
+        var checkifPost = post.description;
```

Comment:
I like the descriptive naming, but I think you can do better here. What IS this thing really?